### PR TITLE
[SDESK-5520] Add elastic manage commands

### PR DESCRIPTION
**Pull request by MarkLark86**

### CODE REVIEW

File: src/server/sams/commands/delete_elastic_index.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        es = get_es(app.config['ELASTICSEARCH_URL'])
+        indices = list(es.indices.get_alias('{}_*'.format(index_prefix)).keys())
+
+        for es_resource in app.data.get_elastic_resources():
+            alias = app.data.elastic._resource_index(es_resource)
+            for index in indices:
+                if index.rsplit('_', 1)[0] == alias:
+                    try:
+                        logger.info('Removing elastic index "{}"'.format(index))
```
Comment (petrjasek) [inline src/server/sams/commands/delete_elastic_index.py:53]: I would probably log via `logger.info('Removing elastic index "%s"', index)`, with format it seems a bit complicated to me,
we should probably pick one or the other to be consistent, any opinion @ride90 @jerome-poisson ?

File: src/server/sams/commands/index_from_mongo.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+        $ python -m sams.manage app:index_from_mongo --from=assets
+        $ python -m sams.manage app:index_from_mongo --all
+
+    """
+
+    option_list = [
+        Option('--from', '-f', dest='collection_name'),
+        Option('--all', action='store_true', dest='all_collections'),
+        Option('--page-size', '-p')
```
Comment (petrjasek) [inline src/server/sams/commands/index_from_mongo.py:40]: I think you can add `type=int, default=500` to avoid that extra default_page_size

**Review by petrjasek — APPROVED** [review]

File: src/server/sams/commands/delete_elastic_index.py (hunk 1 above)
Comment (ride90) [reply src/server/sams/commands/delete_elastic_index.py:53]: I think `format` is a new style and `%` is old, so I would go with the `format`. 
Starting from python3.6 we can use **f-strings**, which are faster and nicer.

**Review by ride90 — APPROVED** [review]

File: src/server/sams/commands/delete_elastic_index.py (hunk 1 above)
Comment (petrjasek) [reply src/server/sams/commands/delete_elastic_index.py:53]: it's not about using `%`, logger supports that message format

Comment (ride90) [reply src/server/sams/commands/delete_elastic_index.py:53]: ahh, right. 
In this case `logger.info('Removing elastic index "%s"', index)` seems more simple.

Are we support python 3.5 for SAMS? If not f-strings is the way to go imo.

Comment (MarkLark86) [reply src/server/sams/commands/delete_elastic_index.py:53]: SAMS uses Superdesk-Core/develop as the application framework, so I would say Python 3.6+ are the supported versions.
Good point @petrjasek, I didn't realise `logger` had `vargs` capability, will update accordingly.

**Pull request merged**